google-cloud-storage/src/main/java/com/google/cloud/storage/HttpRetryAlgorithmManager.java

```diff
@@ -213,6 +213,11 @@ public ResultRetryAlgorithm<?> getForObjectsGet(
     return retryStrategy.getIdempotentHandler();
   }
 
+  public ResultRetryAlgorithm<?> getForObjectsRestore(
+      StorageObject pb, Map<StorageRpc.Option, ?> optionsMap) {
+    return retryStrategy.getIdempotentHandler();
```

Comment:
Missed this on first pass; ObjectRestore is idempotent because it can only succeed once which is similar to Bucket; is that your understanding as well?

Reply:
Yes, restore has the same idempotency as Copy and similar operations